protobuf/psa_raw_key_agreement.proto

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2020 Contributors to the Parsec project.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+syntax = "proto3";
+
+package psa_raw_key_agreement;
+
+import "psa_algorithm.proto";
+
+message Operation {
+  psa_algorithm.Algorithm.KeyAgreement alg = 1;
+  string private_key = 2;
```

Comment:
Maybe call this argument `private_key_name`, to not confuse it with the literal value?

Reply:
Sounds like a reasonable idea, I've updated the book PR for this too 🙂.